polkadot/node/core/backing/src/lib.rs

```diff
@@ -1020,14 +1018,14 @@ async fn handle_active_leaves_update<Context>(
 				"Failed to load implicit view for leaf."
 			);
 
-			return Ok(())
+			return Ok(());
```

Comment:
looks like you're using a different rustfmt version. We usually use a nightly compiler for formatting and configure it with the `.rustfmt.toml` in the repo